src/contract/helpers.js

```diff
@@ -0,0 +1,158 @@
+const { ethers } = require('ethers');
```

Comment:
We've got `src/contract/helpers.js` and `src/lib/helpers.js`. You're throwing various stuff in both. We'll need to group this by functionality and come up with a logical split.